src/main/java/org/spongepowered/common/config/category/TimingsCategory.java

```diff
@@ -33,22 +33,22 @@
 @ConfigSerializable
 public class TimingsCategory extends ConfigCategory {
 
-    @Setting
+    @Setting(comment = "???")
     private boolean verbose = false;
 
-    @Setting
+    @Setting(comment = "???")
```

Comment:
disable timings, might as well disable the module at that point?

Reply:
This is used by the TimingsFactory.
Timings is enabled only if the `Timing` module and this option are both set to true

https://github.com/SpongePowered/SpongeCommon/blob/a865595b36ab95a876c8ebd46b464b23a73b09aa/src/main/java/org/spongepowered/common/relocate/co/aikar/timings/SpongeTimingsFactory.java#L63